custom_json_diff/custom_diff_classes.py

```diff
@@ -160,6 +160,7 @@ def intersection(self, other, title: str = "") -> "BomDicts":
     def generate_counts(self) -> Dict:
         return {
             "filename": self.filename, "components": len(self.components),
+            "frameworks": len([f for f in self.components if f.component_type == "framework"]),
```

Comment:
@prabhu Why are frameworks being counted and no other component types?

Reply:
For my use case, framework is important. Frameworks such as spring usually would be present in pre-build and post-build sboms as well container sboms.

Reply:
@prabhu  Rather than adding them to the counts and doing an extra iteration over the components here, you can just count in the template using the framework variables, which are diff_frameworks_1, diff_frameworks_2, and common_frameworks. These are all passed when generating the report as they get broken down by the to_summary method of the BomDicts class.

![image](https://github.com/AppThreat/custom-json-diff/assets/80227828/f94680e1-ab4d-41ac-b50e-6d08e20a77e9)